src/bitmessagecli.py

```diff
@@ -832,184 +864,194 @@ def sendBrd(fromAddress, subject, message): #sends a broadcast
-        print '     Subject:', outboxMessages['sentMessages'][msgNum]['subject'].decode('base64') #Get the subject
-        print '     Status:', outboxMessages['sentMessages'][msgNum]['status'] #Get the subject
-        
-        print '     Last Action Time:', datetime.datetime.fromtimestamp(float(outboxMessages['sentMessages'][msgNum]['lastActionTime'])).strftime('%Y-%m-%d %H:%M:%S')
+        print '     To:', getLabelForAddress(outboxMessages['sentMessages'][msgNum]['toAddress'])  # Get the to address
+        print '     From:', getLabelForAddress(
+            outboxMessages['sentMessages'][msgNum]['fromAddress'])  # Get the from address
+        print '     Subject:', outboxMessages['sentMessages'][msgNum]['subject'].decode('base64')  # Get the subject
+        print '     Status:', outboxMessages['sentMessages'][msgNum]['status']  # Get the subject
+
+        print '     Last Action Time:', datetime.datetime.fromtimestamp(
+            float(outboxMessages['sentMessages'][msgNum]['lastActionTime'])).strftime('%Y-%m-%d %H:%M:%S')
 
-        if (msgNum%20 == 0 and msgNum != 0):
-            userInput('(Press Enter to continue or type (Exit) to return to the main menu.)').lower() # uInput = 
+        if (msgNum % 20 == 0 and msgNum != 0):
+            userInput('(Press Enter to continue or type (Exit) to return to the main menu.)').lower()  # uInput =
 
     print '\n     -----------------------------------'
-    print '     There are ',numMessages,' messages in the outbox.'
+    print '     There are ', numMessages, ' messages in the outbox.'
     print '     -----------------------------------\n'
 
-def readSentMsg(msgNum): #Opens a sent message for reading
+
+def readSentMsg(msgNum):  # Opens a sent message for reading
     global usrPrompt
     try:
         outboxMessages = json.loads(api.getAllSentMessages())
         numMessages = len(outboxMessages['sentMessages'])
-    except:
+    except BaseException:
         print '\n     Connection Error\n'
         usrPrompt = 0
         main()
-            
+
     print ' '
 
     if (msgNum >= numMessages):
         print '\n     Invalid Message Number.\n'
         main()
 
-    #Begin attachment detection
+    # Begin attachment detection
     message = outboxMessages['sentMessages'][msgNum]['message'].decode('base64')
 
-    while True: #Allows multiple messages to be downloaded/saved
-        if (';base64,' in message): #Found this text in the message, there is probably an attachment.
-            attPos= message.index(";base64,") #Finds the attachment position
-            attEndPos = message.index("' />") #Finds the end of the attachment
-            #attLen = attEndPos - attPos #Finds the length of the message
-
+    while True:  # Allows multiple messages to be downloaded/saved
+        if (';base64,' in message):  # Found this text in the message, there is probably an attachment.
+            attPos = message.index(";base64,")  # Finds the attachment position
```

Comment:
```python
attPos = message.find(";base64,")
if attPos > 0:
    ...
```

Reply:
Not a style issue. I'll let you pursue this change wherever you find it in your own time ;)